scripts/benchmark_bolt.py

```
@@ -0,0 +1,85 @@
+    c.execute(
+        "CREATE TABLE crew_availability (id INTEGER PRIMARY KEY AUTOINCREMENT, crew_id INTEGER NOT NULL, start_time DATETIME NOT NULL, end_time DATETIME NOT NULL)"
+    )
+    c.execute(
+        "CREATE TABLE appliance (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT NOT NULL UNIQUE)"
+    )
+    c.execute(
+        "CREATE TABLE appliance_availability (id INTEGER PRIMARY KEY AUTOINCREMENT, appliance_id INTEGER NOT NULL, start_time DATETIME NOT NULL, end_time DATETIME NOT NULL)"
+    )
+
+    # Insert 50 crew members
+    now = datetime.now()
```

Ruff (0.15.4)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)